### PR TITLE
Court name normalization

### PR DESCRIPTION
**Pull request by ChefAndy**
This should be tested on a larger body of data before it's run.

### CODE REVIEW

**Comment by ChefAndy** [comment]
i merged in develop and all the tests are dying. Gotta see what's up.

File: capstone/fabfile.py [hunk 1]
```diff
@@ -205,7 +206,7 @@ def add_test_case(*barcodes):
     from django.core import serializers
     from tracking_tool.models import Volumes, Reporters, BookRequests, Pstep, Eventloggers, Hollis, Users
     from capdb.storages import ingest_storage
-    from scripts.helpers import parse_xml, copy_file, resolve_namespace, serialize_xml
+    from scripts.helpers import copy_file, resolve_namespace
```
Comment (anastasia) [inline capstone/fabfile.py]: seems like there are two import lines from scripts.helpers
Want to merge them?

**Comment by anastasia** [comment]
Just one comment, otherwise looks good to me!

File: capstone/fabfile.py (hunk 1 above)
Comment (ChefAndy) [reply capstone/fabfile.py]: Nice catch, thanks!

**Pull request merged**